### PR TITLE
[semver:major] First Major Release

### PR DESCRIPTION
**Pull request by ravin-accelq**
First major release of ACCELQ CircleCI Orb

### CODE REVIEW

File: src/@orb.yml [hunk 1]
```diff
@@ -1,15 +1,7 @@
 version: 2.1
 
 description: >
-  Sample orb description
-# What will your orb allow users to accomplish?
-# Descriptions should be short, simple, and informative.
-
-# This information will be displayed in the orb registry and is not mandatory.
+  Use ACCELQ Orb to trigger scheduled CI jobs registered in ACCELQ application.
```
Comment (KyleTryon) [inline src/@orb.yml:4]: Consider a more descriptive explanation of the value and benefits of this integration for users who are not yet familiar with ACCELQ. Keywords will help with search results as well. 

What is ACCELQ and how can adding it to my CircleCI Config add value as a user?

**Review by KyleTryon — CHANGES_REQUESTED** [review]
Hello @accelq + @vivekburman,

In this PR please include the following:

1.  Rename the `example.yml` file to a use-case-based name.
  - From our conversation: Rename the example file to represent the use case this is an example of. Example: `execute_accelq_tests.yml`
2. Delete the un-used `tests` directory
3. Implement integration testing:
  - Lint to test job: https://app.circleci.com/pipelines/github/accelq/accelq-connect-orb/24/workflows/27a50128-ae6f-4487-a5a6-bbe0f185b367/jobs/83
  - Link to integration test config: https://github.com/accelq/accelq-connect-orb/blob/ab6353cf676cab41d4449f5e5a74b19a856a59fe/.circleci/config.yml#L41
  - The test appears to have failed although for some reason the exit code was obscured and the job is showing green
 error:
```
No Agent Found that matches pattern:POC_Agent
(node:54) UnhandledPromiseRejectionWarning: No Agent Found that matches pattern:POC_Agent
(node:54) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:54) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  - Ensure the job properly executes and fails (returns a non-zero exit code) when a failure occurs so the exit may propagate correctly.
```

File: src/@orb.yml (hunk 1 above)
Comment (ravin-accelq) [reply src/@orb.yml:4]: 1. Already have renamed example.yml to execute_accelq_tests.yml
2. Done
3. Integration testing is correct. Its intentionally filled with wrong values. Such that no one triggers the actual job in accelq side and eat up resources. Further the error means agent is not availbale in accelq to run a job. The indication that integration test is working is that the job was successful to login to accelq.
4. I believe that is enough information. As the ci job is a technical term to accelq. So anyway customers need to have basic understanding before using it.

**Review by KyleTryon — APPROVED** [review]
Thank you for the explanation 👍

**Comment by ravin-accelq** [comment]
So can we merge @KyleTryon ?

**Comment by KyleTryon** [comment]
> So can we merge @KyleTryon ? 

Let's ship it!

**Comment by ravin-accelq** [comment]
> > So can we merge @KyleTryon ?
> 
> Let's ship it!


DONE

**Pull request merged**